.circleci/config.yml

```diff
@@ -157,17 +157,17 @@ commands:
       - checkout
       - restore_cache:
           keys:
-            - v${CACHE_VERSION}-{{ arch }}-dependencies-{{ checksum "package.json" }}-{{ checksum "package/requirements.txt" }}-{{ checksum "package/test_requirements.txt" }}
+            - v0-dependencies-{{ checksum "package.json" }}-{{ checksum "package/requirements.txt" }}-{{ checksum "package/test_requirements.txt" }}
```

Comment:
```suggestion
            - v0-dependencies-{{ arch }}-{{ checksum "package.json" }}-{{ checksum "package/requirements.txt" }}-{{ checksum "package/test_requirements.txt" }}
```

Reply:
Let's keep the `arch` in there for now (same change for all other instances in the file).